git_pw/patch.py

```diff
@@ -329,6 +329,25 @@ def update_cmd(patch_ids, commit_ref, state, delegate, archived, fmt):
     is_flag=True,
     help='Include patches that are archived.',
 )
+@click.option(
+    '--since',
+    'since',
+    metavar='SINCE',
+    help=(
+        'Show only patches since a given date. Should be in ISO 8601 '
+        'format, e.g. \'2023-05-09T00:00:00\''
+    ),
+)
+@click.option(
+    '--before',
+    'before',
+    metavar='BEFORE',
+    help=(
+        'Show only patches before a given date. Should be in ISO 8601 '
+        'format, e.g. \'2023-05-09T00:00:00\''
+    ),
+)
```

Comment:
Unless GitHub is rendering this funny, these are duplicates and should be removed. The `@utils.date_options()` decorator is doing this for you now.